### PR TITLE
reverting ledger-dist folder to work dir and work dir now in target dir

### PR DESCRIPTION
**Pull request by BrianCorbin**
to resolve conversation from https://github.com/mobilecoinfoundation/mobilecoin/pull/1943

### CODE REVIEW

File: tools/local-network/local_network.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 MOB_RELEASE = os.getenv('MOB_RELEASE', '1')
 CARGO_FLAGS = '--release'
 TARGET_DIR = 'target/release'
-WORK_DIR = '/tmp/mc-local-network'
+WORK_DIR = os.path.join(TARGET_DIR, '/tmp/mc-local-network')
```
Comment (eranrund) [inline tools/local-network/local_network.py:34]: This should not have `/tmp/` there.

Comment (eranrund) [reply tools/local-network/local_network.py:34]: I also believe it should be an absolute path so you probably need to do something like `os.path.abspath(os.path.join(PROJECT_DIR, TARGET_DIR, 'mc-local-network'))`.
Please ensure this actually runs and everything works.

Comment (eranrund) [reply tools/local-network/local_network.py]: yes, thats where it should end up at. I am not sure a relative path would work unless everything is executed from inside PROJECT_DIR (which may or may not be the case)

Comment (BrianCorbin) [reply tools/local-network/local_network.py]: oh, maybe not!

**Review by eranrund — APPROVED** [review]

**Review by garbageslam — APPROVED** [review]

**Pull request merged**